ppr-ui/src/store/store.ts

```diff
@@ -515,6 +517,14 @@ export const useStore = defineStore('assetsStore', () => {
     return state.value.mhrRegistration.ownLand
   })
 
+  // MHR Unit Notes
+  const getMhrRegistrationUnitNotes = computed<MhrRegistrationUnitNoteIF[]>(() => {
+    return state.value.mhrRegistration.notes
+  })
```

Comment:
Same as above, unit notes are their own filing.
https://amer-api-partner61-demo.apigee.io/docs/mhrproxy/1/routes/mhr/api/v1/notes/%7BmhrNumber%7D/post

Reply:
This is to get the Notes from the MHR Registrations `GET /mhr/api/v1/registrations/{mhrNumber}`.

Reply:
The state `mhrRegistration` is model for the registration filing itself.

Reply:
It could* be stored anywhere for sure and It definitely is a part of the GET due to their coupling to the Registration Record 

Though i feel like the unitNotes(the notes history) should be standalone, in the same manner you did the unitNote (filing state) though due to the fact they are standalone filings and they are mostly just an association/link to an MHR, not part of the original mhrRegistration.